client/lib/analytics/ad-tracking.js

```diff
@@ -787,28 +832,47 @@ function recordProduct( product, orderId ) {
 		// Google AdWords
 		if ( isAdwordsEnabled ) {
 			if ( window.google_trackConversion ) {
-				window.google_trackConversion( {
-					google_conversion_id: isJetpackPlan
-						? ADWORDS_CONVERSION_ID_JETPACK
-						: ADWORDS_CONVERSION_ID,
-					google_conversion_label: isJetpackPlan
-						? TRACKING_IDS.googleConversionLabelJetpack
-						: TRACKING_IDS.googleConversionLabel,
-					google_conversion_value: product.cost,
-					google_conversion_currency: product.currency,
-					google_custom_params: {
-						product_slug: product.product_slug,
-						user_id: userId,
-						order_id: orderId,
-					},
-					google_remarketing_only: false,
-				} );
+				if ( product.is_free_signup ) {
+					window.google_trackConversion( {
```

Comment:
Would it be cleaner to not duplicate the `window.google_trackConversion( {` block and set in advance two local variables like `googleConversionId` and `googleConversionLabel` based on `isJetpackPlan` and `product.is_free_signup` and pass them to the one `window.google_trackConversion( {`?

Reply:
Good suggestion. Yes, can do.

Reply:
Done. Thank you 😄